Formula/ksync.rb

```diff
@@ -0,0 +1,30 @@
+class Ksync < Formula
+  desc "Sync files between your local system and a kubernetes cluster"
+  homepage "https://ksync.github.io/ksync/"
+  url "https://github.com/ksync/ksync.git",
+    :tag      => "0.4.5",
+    :revision => "9f40bf134329814a57e1a58d73b84761a2b06c73"
+  sha256 "bd47c5b5418bfc6d2f08abe855a52d4189f8bf380747cdd4e5c1a3ca0a87498a"
+  license "Apache-2.0"
+  depends_on "go" => :build
+
+  def install
+    # Set some stamps and build local CLI binary
+    time = `date +%Y%m%dT%H%M%S`.chomp
+    goversion = "go#{Formula["go"].version}"
+    commit = Utils.safe_popen_read("git", "rev-parse", "--short", "HEAD").chomp
+    system "go", "build", "-ldflags",
+              "-w -X github.com/ksync/ksync/pkg/ksync.GitCommit=#{commit} \
+              -X github.com/ksync/ksync/pkg/ksync.GitTag=#{version} \
+              -X github.com/ksync/ksync/pkg/ksync.BuildDate=#{time} \
+              -X github.com/ksync/ksync/pkg/ksync.VersionString=Homebrew \
+              -X github.com/ksync/ksync/pkg/ksync.GoVersion=#{goversion}",
+              *std_go_args,
+              "github.com/ksync/ksync/cmd/ksync"
+  end
+
+  test do
+    expected = 'level=fatal msg="Unable to get config for context (). Does \'kubectl --context= cluster-info\' work?"'
+    assert_equal "#{expected}", shell_output("#{bin}/ksync init --local --log-level debug | awk '{print substr($0, index($0,$2))}'").split("\n").pop
```

Comment:
This should be enough.
```suggestion
    expected = "level=fatal msg=\"Unable to get config for context (). Does 'kubectl --context= cluster-info' work?\""
    assert_equal "#{expected}", shell_output("#{bin}/ksync init --local --log-level debug")
```

Reply:
`awk` (or some method of string substitution) is needed to trim the timestamp off the front of the error message. Since the audit test seems to dislike `awk`, I'm looking at using native ruby methods.

Reply:
Just use `assert_match` then. It'll take a regex (or just match a string anywhere)